src/jit/inline.cpp

```diff
@@ -1265,6 +1275,15 @@ void InlineStrategy::DumpXml(FILE* file, unsigned indent)
         return;
     }
 
+    // Dump header
+    if (!s_DumpXmlHeader)
+    {
+        fprintf(file, "<?xml version=\"1.0\"?>\n");
+        fprintf(file, "<InlineForest>\n");
+        fprintf(file, "<Methods>\n");
+        s_DumpXmlHeader = true;
+    }
+
```

Comment:
Minor point, but "DumpXmlHeader sounds like an imperative - like it should be dumped. Maybe s_DumpedXmlHeader?